src/Microsoft.AspNetCore.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -129,7 +129,15 @@ public OAuthHandler(HttpClient backchannel)
                 properties.StoreTokens(authTokens);
             }
 
-            return AuthenticateResult.Success(await CreateTicketAsync(identity, properties, tokens));
+            try
+            {
+                var ticket = await CreateTicketAsync(identity, properties, tokens);
+                return AuthenticateResult.Success(ticket);
+            }
+            catch(Exception ex)
```

Comment:
I'm with @Tratcher. I don't think `RemoteRemoteAuthenticateAsync` should throw. It returns `AuthenticateResult` which has a exception property on it. That indicate the method should handle the exception itself. On the other hand `CreateTicketAsync` returns `AuthenticationTicket` which doesn't have a property to carry error message.